### DISCUSSION TITLE
Would it make sense to move the UserPlugins documentation?

### DISCUSSION THREAD

**Discussion by keckler** [terrapower/armi#934] [General]
A few times recently I have found myself looking for the documentation on UserPlugins. UserPlugins are an awesome feature, but I find that I can never find the docs just by perusing the doc tree.

They are currently located under "Tutorials":
https://terrapower.github.io/armi/tutorials/making_your_first_app.html#bonus-ad-hoc-userplugins

IMO, it might make more sense to instead have these docs located under the "User Docs::Inputs" section:
https://terrapower.github.io/armi/user/inputs/index.html

My thinking is that a UserPlugin is really just an input file, similar to the custom shuffle logic. And users of the code looking for instructions typically look in the user docs, not in the tutorials, when they are looking for the syntax of specific files.

**Comment by john-science** [comment]
You want `UserPlugin` docs moved from "Tutorials" to "User Docs"?

That seems strange to me. It seems like if we bury `UserPlugin`s further down in the docs, fewer people will see that they exist.

We can try it if you like though.

**Reply by keckler** [reply]
Well this is a discussion! So that's just my suggestion. I basically don't ever look in the tutorials section of the docs, I always jump to the user docs or the API docs.

To me, "tutorial" is like "I just want an example of something", not "give me the specs of this particular feature".

I don't really care, I just thought it made more sense that way.

**Reply by john-science** [reply]
Like I said, I'm not going to bat over this.  I think ARMI has _pretty good_ docs. But there is enough stuff there that I've had trouble finding things in the past, too. 

The search bar on our docs is nice, but it frequently hinders me that it searches the entire API as well as the docs.  So, sure, it's not perfect.

I wonder if the docs could be organized better.
